cmd/expiration-mailer/main.go

```diff
@@ -116,14 +116,20 @@ func (m *mailer) updateCertStatus(serial string) error {
 	tx, err := m.dbMap.Begin()
 	if err != nil {
 		m.log.Err(fmt.Sprintf("Error opening transaction for certificate %s: %s", serial, err))
-		tx.Rollback()
+		rollbackErr := tx.Rollback()
+		if rollbackErr != nil {
+			m.log.Err(fmt.Sprintf("Also, an error occured when rolling back the transaction: %s", rollbackErr))
```

Comment:
Maybe restructure this as I suggest above, so both errors can be logged together?